client/blocks/comments/form.scss

```diff
@@ -89,6 +84,22 @@
 	}
 }
 
+.comments__comment-list {
+	.comments__form {
+		button {
+			margin: 16px 0 -19px;
+		}
+	}
+}
+
```

Comment:
I'd remove this and leave the `-19px` as a default in the original `margin` statement above.

Reply:
My reasoning was that the -19px margin is saying "I expect to be placed above something with excessive top-margin", which is a statement about the environment the component is placed in, not the component itself.  But I can make this change.

Reply:
I was thinking about that as well. I could go either way on it. I see what you mean for sre.